docs/dev/konnector.md

```diff
@@ -42,24 +41,11 @@ Then, write your code into `konnectors.js` and build the application, running `y
 
 ### Collector structure
 
-Basically, a collector is just an object with some required fields:
+Basically, a connector is just an object passed to baseKonnector.createNew:
 
- - `name`: the displayed name of the collector;
```

Comment:
My explanation was not complete. Not all the attributes are in the manifest, the other ones are defined in cozy-collect (colors and things like that), waiting for a real marketplace for connectors I think). The attributes you describe in the manifest part are OK.